acrobat/scripts/scripts.js

```diff
@@ -151,24 +151,28 @@ const CONFIG = {
 
 (async function loadPage() {
   // Fast track the widget
-  const widgetBlock = document.querySelector('.dc-converter-widget');
-  if (widgetBlock) {
-    widgetBlock.removeAttribute('class');
-    widgetBlock.id = 'dc-converter-widget';
-    const { default: dcConverter } = await import('../blocks/dc-converter-widget/dc-converter-widget.js');
-    dcConverter(widgetBlock);
-  }
+  (async () => {
+    const widgetBlock = document.querySelector('.dc-converter-widget');
+    if (widgetBlock) {
+      widgetBlock.removeAttribute('class');
+      widgetBlock.id = 'dc-converter-widget';
+      const { default: dcConverter } = await import('../blocks/dc-converter-widget/dc-converter-widget.js');
+      dcConverter(widgetBlock);
+    }
+  })();
+
+  // Setup CSP
+  (async () => {
+    if (document.querySelector('meta[name="dc-widget-version"]')) {
+      const { default: ContentSecurityPolicy } = await import('./contentSecurityPolicy/csp.js');
```

Comment:
perhaps csp could be moved to the deferred section with dcLana, but try this for now?